src/assets/station-templates/keba.station-template.json

```diff
@@ -43,6 +43,11 @@
         "key": "WebSocketPingInterval",
         "readonly": false,
         "value": "60"
+      },
+      {
+        "key": "ReserveConnectorZeroSupported",
```

Comment:
Is that key part of the OCPP 1.6 standard keys?
If yes, please add it also to the README.md `OCPP-J standard parameters supported` section.